west.yml

```diff
@@ -216,7 +216,7 @@ manifest:
       path: modules/lib/open-amp
     - name: openthread_telink_lib
       url: https://github.com/telink-semi/openthread_telink_lib
-      revision: 80645ba0b4b589afd8c0712cb1a01091f4b35ab8
+      revision: 84e12fd24613b2871c47378c39c2f44447f528ba
```

Comment:
@damien0x0023 this hash is from tl321x_skeleton branch of openthread_telink_lib
Please merge this branch into main and update hash after

Reply:
I see.
Just create the PR there.